docs/versioned-plugins/outputs/elasticsearch-v9.3.0.asciidoc

```diff
@@ -98,6 +98,12 @@ happens, the problem is logged as a warning, and the event is dropped. See
 
 beta[]
 
+[IMPORTANT]
+To use Index Lifecycle Management feature, please upgrade this plugin to the latest available version
+using the Logstash plugin upgrader:
+`bin/logstash-plugin upgrade logstash-output-elasticsearch`
+
```

Comment:
let's add a note that we should see the plugin be upgraded to at least version 9.3.1